valhalla/thor/pathalgorithm.h

```diff
@@ -89,6 +90,23 @@ class PathAlgorithm {
     return has_ferry_;
   }
 
+  /**
+   * Set the not_thru_pruning_
+   * @param pruning  set the not_thru_pruning_ to pruning value.
+   *                 only set on the second pass
```

Comment:
this is set to true on first pass for bidirectional, so this comment line is incorrect

Reply:
I would consider moving the comments on not_thru pruning from bidirectional_astar.cc into here.